### ISSUE TITLE
Dinput 360 controller not detected by current version.

### ISSUE THREAD

**Issue by LuisCondeVela** [Stents-/XOutput#24]
I trying to set it with a Dinput 360 controller, but its just not being detected, XOutput is just gray with the exclusivity checkbox and the Star at the end and nothing else.

the older release 0.1 did detected it and even map it automatically, but its just keeps crashing on Start.

Windows 10 64 bits, updated to date, i did installed ScpDriver and SlimDX runtime. Runned all as Administrator.

Thank you and good job.

**Comment by ericlbarrett** [comment]
Are you using a xbox 360 controller? XInput ignores 360 controllers. Though I don't know why v0.1 would detect it.

**Comment by LuisCondeVela** [comment]
yes i have a 360 controller in Direct Input mode with custom drivers, XBCD, i use them so i can set dead zones and move the trigger to separate axis.

well, i guess ill look for other solutions to get it to work with UWP games. Thanks

**Comment by joao678** [comment]
I managed to fix this. Check this out!
https://github.com/joao678/XOutput-XBCD/releases/tag/v0.12

**Comment by Squall-Leonhart** [comment]
im curious what you fixed, since 0.11 works fine on xbcd o.O
